lib/Sema/DerivedConformanceEquatableHashable.cpp

```diff
@@ -196,7 +196,8 @@ deriveEquatable_enum_eq(TypeChecker &tc, Decl *parentDecl, EnumDecl *enumDecl) {
   //   case A, B, C
   //
   //   @derived
-  //   func ==(a: SomeEnum<T...>, b: SomeEnum<T...>) -> Bool {
+  //   @_implements(Equatable, ==(_:_:))
+  //   func _DerivedEnumEquals(a: SomeEnum<T...>, b: SomeEnum<T...>) -> Bool {
```

Comment:
I think it's already marked implicit, so most people won't see it. There are just a few tests that try to print implicit declarations as well.